src/components/RequestModal/MovieRequestModal.tsx

```diff
@@ -23,12 +23,15 @@ const messages = defineMessages({
   requesttitle: 'Request {title}',
   request4ktitle: 'Request {title} in 4K',
   edit: 'Edit Request',
+  editAndApprove: 'Edit & Approve Request',
```

Comment:
```suggestion
  editAndApprove: 'Approve Request',
```
I think it's expected that any modifications to the advanced request settings made by an admin in the modal will be applied to the approved request.